src/window.cpp

```diff
@@ -179,27 +189,78 @@ window::draw_desktops ()
   for (const auto &desktop : this->desktops)
     {
       xcb_put_image (window::c, XCB_IMAGE_FORMAT_Z_PIXMAP,
-                     this->xcb_id,                  /* Pixmap to put image on */
-                     window::gc,                    /* Graphic context */
-                     desktop.width, desktop.height, /* Dimensions */
+                     this->xcb_id, /* Pixmap to put image on */
+                     window::gc,   /* Graphic context */
+                     desktop.pixmap_width, desktop.pixmap_height,
```

Comment:
Do you think we need to get reed of /* Dimensions */ comment?

Reply:
What does the name `height` of the variable tell you?

Do you really need a dedicated comment that will tell you that `height` is
height?

Reply:
I guess I want the format to be same for every line
a // comment for a
b
c//comment for c
seems strange